smartypants.go

```diff
@@ -398,13 +400,14 @@ func NewSmartypantsRenderer(flags Extensions) *SPRenderer {
 	return &r
 }
 
-func (sr *SPRenderer) Process(text []byte) []byte {
+// Process is the entry point of the Smartypants renderer.
+func (smrt *SPRenderer) Process(text []byte) []byte {
```

Comment:
Yeah, golint complained about inconsistent receiver naming (all the rest funcs have `smrt`). I like `sr` better, but chose less intrusive change for this commit.